Cargo.toml

```diff
@@ -37,7 +37,7 @@ axum-server = { version = "0.5.1", optional = true, features = ["rustls", "rustl
 base64 = { version = "0.21.2", optional = true }
 bitvec = "1.0"
 bytes = "1.4"
-clap = { version = "4.0", optional = true, features = ["derive"] }
+clap = { version = "4.3.2", optional = true, features = ["derive"] }
```

Comment:
clap build started failing suddenly with weird errors, had to upgrade it to unblock